README.md

```diff
@@ -9,34 +9,37 @@ Get the main repository:
 git clone https://github.com/mean00/avidemux2.git
 ```
 
-Get submodules:
+It needs an additional i18n (translation files) submodule. You can get it with:
 ```
 git submodule update --init --recursive
 ```
 
-## Debian / Ubuntu and derivatives
-
-Required packages:
-```
-sudo apt install gcc build-essential cmake pkg-config yasm libsqlite3-dev  qtbase5-dev qttools5-dev-tools wget
-```
-
-Recommended packages:
+The application can be built with the following convenience script (it will check and install all dependencies):
```

Comment:
I'm sory, I was unclear. I meant primarily `createDebFromSourceUbuntu.bash` for Ubuntu and `createRpmFromSourceFedora.bash` for Fedora. I recommend to use them just to setup the environment (`--deps-only`) and perform the actual build in a separate step directly using `bootStrap.bash` with desired options.

All the appImage scripts are more for official distribution, the older ones were tailored to specific environment present in official build nodes. The latest "Minimal" script should work on vanilla Buster (else there is a bug), and whoever on Debian Buster prefers to generate an appImage can use it.